### ISSUE TITLE
No mp3-streaming over plugin, after Update to 2.0

### ISSUE THREAD

**Issue by Matt-Eagle-Pi** [petro-kushchak/homebridge-homepod-radio#34]
Hi Petro,

since the update to the new 2.0-version, the mp3-stream to my Homepod mini doesn't work within homekit-automation. 
Do I have to change a setting?

The manual-start of the shortcut on my iphone works, but now shows a messagebox with the Error: "Could not connect to Server". The mp3 is playing after a few secands. 

Via Homekit, the stream is not triggerd properly. Here is the log:  

[23.2.2023, 18:16:13] [@petro-kushchak/homebridge-homepod-radio] [Filmabend] Started streaming http://192.168.0.104:8585/play/kino.mp3
Error: This callback function has already been called by someone else; it can only be called one time.
    at /var/lib/homebridge/node_modules/homebridge/node_modules/hap-nodejs/src/lib/util/once.ts:12:13
    at /var/lib/homebridge/node_modules/@petro-kushchak/homebridge-homepod-radio/src/lib/homebridgeCallbacks.ts:19:34
    at processTicksAndRejections (node:internal/process/task_queues:95:5)

Greetings from Germany!

Matt

**Comment by petro-kushchak** [comment]
Hi, which version of tvOS you are running on the HomePod mini? I've noticed that with latest tvOS 16.4 beta both web streaming and file streaming is not reliable... 

Also could you please check this beta version of the plugin @petro-kushchak/homebridge-homepod-radio-2.0.9? It have fixes which should make audio streaming more resilient, also you can add "audio switch" accessory which will stream file when "ON" (instead of using HTTP request "/play/<mp3-file>")

Thanks, Petro

**Comment by Matt-Eagle-Pi** [comment]
Hi Petro.

The TvOs is 16.3.2. I'll check the beta version later.

Matt

**Comment by Matt-Eagle-Pi** [comment]
Hit Petro.

Sry, but i find only the Version 2.0.0. The switch-setting, i've tested earlier with this version, but the stream doesn't work...
Could you send me a link to the version 2.0.9?

Thanks, 
Matt

**Comment by petro-kushchak** [comment]
Hi, follow steps from this doc about plugin beta testing with Homebridge: https://github.com/homebridge/homebridge/wiki/How-to-Install-Alternate-Plugin-Versions

**Comment by Matt-Eagle-Pi** [comment]
Hi Petro. 

Now, I installed the beta-version 2.0.9 and set the audio-button-settings, instead of the radio-settings. Before that, I've deleted the "old" speakers in Homekit. The manual stream via "shortcuts" now works without error-message, but the stream via homekit, still doesn't start. Do I have to change the stream-url in the audio-button-setting, too?

Thanks for your support so far!

Matt
